src/coreclr/utilcode/stresslog.cpp

```diff
@@ -185,13 +188,19 @@ static LPVOID CreateMemoryMappedFile(LPWSTR logFilename, size_t maxBytesTotal)
     }
 
     size_t fileSize = maxBytesTotal;
+    // TODO: PAL does not support mapping larger than 32 bit
```

Comment:
It seems that we can fix it in PAL easily. The `ftruncate` Unix function that ends up getting called has `off_t` type of the argument, which is 64 bit even for our 32 bit builds now. So it seems that modifying the type of the `NewSize`  argument of the `MAPGrowLocalFile` to be `off_t` and updating the `CorUnix::InternalCreateFileMapping` to combine the low / high parts into an `off_t` value and using it in that method instead of just `dwMaximumSizeLow` would do the trick.

Reply:
And remove the `if (0 != dwMaximumSizeHigh)` where we bail out too.